scripts/measure_baseline.py

```
@@ -0,0 +1,198 @@
+    if ns < 1_000:
+        return f"{ns:.2f} ns"
+    elif ns < 1_000_000:
+        return f"{ns / 1_000.0:.2f} µs"
+    elif ns < 1_000_000_000:
+        return f"{ns / 1_000_000.0:.3f} ms"
+    else:
+        return f"{ns / 1_000_000_000.0:.3f} s"
+
+
+def run_command(cmd: List[str]) -> str:
+    res = subprocess.run(cmd, capture_output=True, text=True, check=True)
```

Ruff (0.16.3)

[error] `subprocess` call: check for execution of untrusted input

(S603)